node/runtime/src/lib.rs

```diff
@@ -524,12 +525,23 @@ impl_runtime_apis! {
 		}
 	}
 
-	impl consensus_aura::AuraApi<Block, AuraId> for Runtime {
-		fn slot_duration() -> u64 {
-			Aura::slot_duration()
+	impl babe_primitives::BabeApi<Block> for Runtime {
+		fn startup_data() -> babe_primitives::BabeConfiguration {
+			babe_primitives::BabeConfiguration {
+				median_required_blocks: 1000,
+				slot_duration: Babe::slot_duration(),
+				threshold: core::u64::MAX,
```

Comment:
probably what we should really be parameterizing is the `c` value from `phi_c`, since the threshold is per-validator.